pkg/asset/agent/manifests/nmstateconfig.go

```diff
@@ -124,15 +125,15 @@ func (n *NMStateConfig) Load(f asset.FileFetcher) (bool, error) {
 }
 
 func (n *NMStateConfig) finish() error {
+	if !common.AgentConfigAvaialble {
```

Comment:
I think that's another example of cross-assets validation (ie a validation rule that needs two or more inputs coming from different assets). In such case the usage of global variables IMHO is highly discouraged, whereas it would be recommended to push up the validation rule into _another_ asset depending on all the assets providing the inputs for the validation rule (a similar approach has been taken for AgentManifest for example, see [here](https://github.com/openshift/installer/blob/9b36612243bd08dbe5c32cd851741ffb5e9b9b61/pkg/asset/agent/manifests/agent.go#L125))